packages/fx-core/resource/package.nls.json

```diff
@@ -1131,7 +1131,7 @@
   "driver.spfx.info.tenantAppCatalogCreated": "SharePoint tenant app catalog %s created, wait for a few minutes to be active.",
   "driver.spfx.warn.noTenantAppCatalogFound": "No tenant app catalog found, retry: %s",
   "driver.spfx.error.failedToGetAppCatalog": "Cannot get app catalog site url after creation. You may need wait a few minutes and retry.",
-  "driver.spfx.error.noValidAppCatelog": "There is no valid app catalog in your tenant. You can update the property 'createAppCatalogIfNotExist' in app.yml to true if you want us to create it for you or you can create it by yourself.",
+  "driver.spfx.error.noValidAppCatelog": "There is no valid app catalog in your tenant. You can update the property 'createAppCatalogIfNotExist' in yml file to true if you want us to create it for you or you can create it by yourself.",
```

Comment:
us -> Teams Toolkit

yml file: suggestion - still prefer to quote the full names, if there are multiple yml files, we at least want them to figure out the exact file to make it.

Reply:
Updated